.docker/Dockerfile.alpine

```diff
@@ -0,0 +1,36 @@
+FROM node:12.16-alpine3.9
+
+RUN apk add --no-cache python make g++ libc6-compat ttf-dejavu
```

Comment:
```suggestion
RUN apk add --no-cache python3 make g++ libc6-compat ttf-dejavu
```

Reply:
any way we can clean up the image by removing these packages after running `npm install`?

Reply:
hm... I'm not sure.  Possibly? I'm not sure what that would effect though.  especially the libc6 stuff as sharp I know uses I think so does a few other libraries